Neos.ContentRepository.Core/Classes/Projection/ProjectionsAndCatchUpHooks.php

```diff
@@ -13,7 +13,8 @@
      * @param array<class-string<ProjectionInterface<ProjectionStateInterface>>, CatchUpHookFactories> $catchUpHookFactoriesByProjectionClassName
      */
     public function __construct(
-        public Projections $projections,
+        public ContentRepositoryReadModelProjection $readModelProjection,
```

Comment:
I don't really like that distinction here.. In this case we should not make a difference between the individual projections IMO